env.example

```diff
@@ -2,4 +2,4 @@ API_KEY=
 API_URL=https://api-dev.mbtace.com/
 SWIFTLY_AUTHORIZATION_KEY=
 SWIFTLY_REALTIME_VEHICLES_URL=https://api.goswift.ly/real-time/mbta-bus/vehicles?verbose=true
-export TRIP_UPDATES_URL=https://s3.amazonaws.com/mbta-busloc-s3/staging/TripUpdates_enhanced.json
+export TRIP_UPDATES_URL=https://cdn.mbta.com/realtime/TripUpdates.json
```

Comment:
question: Just checking, do you get all the same data we need from this endpoint? I _think_ that `TripUpdates_enhanced.json` might have extra fields that `TripUpdates.json` doesn't, but I don't know if we are actually using any of them if that's true.

Reply:
oh, oops, meant to get the enhanced one.  thanks!